tests/rules/test_document_start.py

```diff
@@ -102,3 +102,65 @@ def test_directives(self):
                    '---\n'
                    'doc: 2\n'
                    '...\n', conf)
+
+    def test_empty_lines(self):
+        conf = ('document-start:\n'
+                '  present: true\n'
+                '  max-empty-lines-after: 0\n')
+
+        self.check('%YAML 1.2\n'
+                   '---\n'
+                   'doc: ument\n', conf)
+
+        self.check('%YAML 1.2\n'
+                   '---\n'
+                   '\n'
+                   'doc: ument\n', conf, problem=(4, 1))
```

Comment:
To be consistent with other tests, I propose to remove some empty lines to group test cases:
```python
conf = ...
self.check(...)
self.check(...)

conf = ...
self.check(...)
self.check(...)
```

Reply:
Done.